app/controllers/cases/case/subcases/subcase/new-submission.js

```diff
@@ -222,7 +231,7 @@ export default class CasesCaseSubcasesSubcaseNewSubmissionController extends Con
     this.notificationComment = newNotificationData.notificationComment;
   };
 
-  createSubmission = dropTask(async () => {
+  createSubmission = dropTask(async (_meeting) => {
```

Comment:
This is either a meeting from the proposableAgendasModel or a pointer event from the "on click" from the confirmationModal

Reply:
fixed by checking properties